app/google-services.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Do we want this file checked into version control since it contains API keys and such?